theme/static/js/hs_file_browser.js

```diff
@@ -1243,6 +1244,78 @@ function updateNavigationState() {
     $("#fb-move-up").toggleClass("disabled", !(getCurrentPath().path.length || getCurrentPath().hasOwnProperty("aggregation")));    // The root path is an empty string
 }
 
+function isSelected(fullPaths) {
+    const deleteList = $("#fb-files-container li.ui-selected");
+    let filesToDelete = [];
+
+    if (deleteList.length) {
+        for (let i = 0; i < deleteList.length; i++) {
+            const item = $(deleteList[i]);
+            const respath = item.attr("data-url");
+            const pk = item.attr("data-pk");
+            if (respath && pk) {
+                const parsed_path = respath.split(/contents(.+)/).filter(function(el){return el})
+                filesToDelete.push(parsed_path[parsed_path.length-1])
+            } else {
+                if (!isVirtualFolder(item.first())) { // Item is a regular folder
+                    let folderName = item.children(".fb-file-name").text();
+                    let folder_path = getCurrentPath().path.concat(folderName);
+                    filesToDelete.push('/' + folder_path.join('/'))
+                }
+            }
+        }
+
+        // maximum matches with duplicates (based on parent folder match or top level filename match)
+        let matches = fullPaths.map(function(i1) {
+            return filesToDelete.filter(function(i2) {
+                if (i1.split('/')[1] === i2.split('/')[1]) {
+                    return i2
+                }
+            })
+        }).flat()
+        return matches
+    }
+}
+
+function countSelectedExternalReferences(fullPaths) {
```

Comment:
Can this be removed?  I don't see anything using it.

Reply:
Removed